font-test-data/src/ift.rs

```diff
@@ -0,0 +1,412 @@
+use read_fonts::types::Tag;
```

Comment:
Suggest a brief intro comment, e.g. `//! Test data for incremental transfer [link]